app/helpers/recipes_helper.rb

```diff
@@ -31,4 +31,11 @@ def recipe_ingredient_ids(recipe)
   def available_meal_plans_dropdown(user, recipe)
     user.meal_plans.future - recipe.meal_plans
   end
+
+  private
+
+  def first_time_is_this_week?(recipe)
```

Comment:
This is the new feature that I wanted to build. I could not find a font awesome icon that would work, so i switched to material icons and this is what started the whole thing. 😆

I could have made a smaller pr with just this method, but come on, where's the fun in that? 😬